indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -33,7 +36,14 @@ class RHEmailTemplateList(RHManageAbstractsBase):
     """Display list of e-mail templates."""
 
     def _process(self):
-        return jsonify_template('events/abstracts/management/notification_tpl_list.html', event=self.event_new)
+        self.commit = False
```

Comment:
Why is that needed here? `create_mock_abstract` doesn't create any real objects anymore so there won't be anything committed that shouldn't be committed anyway.